src/transformers/modeling_utils.py

```diff
@@ -781,46 +780,49 @@ def prune_heads(self, heads_to_prune: Dict[int, List[int]]):
 
         self.base_model._prune_heads(heads_to_prune)
 
-    def save_pretrained(self, save_directory: Union[str, os.PathLike]):
+    def save_pretrained(
+        self,
+        save_directory: Union[str, os.PathLike],
+        is_main: bool = True,
```

Comment:
Could we maybe just rename the argument to `save_config` ? This would be much more readable to me and also would make sense to allow a user to optionally not save the config.